plugin/build.gradle

```diff
@@ -184,10 +187,23 @@ integTest {
     systemProperty "user", System.getProperty("user")
     systemProperty "password", System.getProperty("password")
 
+    // Only tenant aware test if set
+    if (System.getProperty("tests.rest.tenantaware") != null) {
```

Comment:
don't we need to check `true` too?

Reply:
Yay, you're looking at this PR! :)

Yes, we do.  Most of the `build.gradle` right now is a work in progress; we need to figure out a way to test on all the different remote stores but they also require a lot more setup (docker, ddb, etc.).   Still need to do some cleanup at the end of all the tests as well (since the "wipe all indices" doesn't work remotely).